DuggaSys/courseed.js

```diff
@@ -459,119 +459,3 @@ function htmlFix(text){
 
     return text.replace(/[&<>"']/g, function(m) { return map[m]; });
 }
-
-/*Validates all coursenames*/ 
-function validateCourseName(nameid, dialogid) {
-	var Name = /[a-zA-Z]+/;
-	//regex for only letters
-	var name = document.getElementById(nameid);
-	var x = document.getElementById(dialogid);
-
- //if input contains only letters
-	if (name.value.match(name)) {
-		name.style.borderColor = "#383";
-		name.style.borderWidth = "2px";
-		x.style.display = "none";
-		if (nameid === 'ncoursename') {
-			window.bool3 = true;
-		}
-		if (nameid === 'coursename') {
-			window.bool4 = true;
-		}
-
-	} else {
-
-		name.style.borderColor = "#E54";
-		x.style.display = "block";
-		name.style.borderWidth = "2px";
-
-		if (nameid === 'ncoursename') {
-			window.bool3 = false;
-		}
-		if (nameid === 'coursename') {
-			window.bool4 = false;
-		}
-
-
-	}
-}
-/*Validates all coursecodes*/ 
-function validateCourseCode(codeid, dialogid) {
-	var Code = /^[a-zA-Z]{2}\d{3}[a-zA-Z]{1}$/;
-	//regex for 2 letters, 3 digits, 1 letter
-
-	var code = document.getElementById(codeid);
-	var x2 = document.getElementById(dialogid);
-
- 	//if input is 2 letters, 3 digits, 1 letter
-	if (code.value.match(Code)) {
-		code.style.borderColor = "#383";
-		code.style.borderWidth = "2px";
-		x2.style.display = "none";
-		if (codeid === 'ncoursecode') {
-			window.bool = true;
-		}
-		if (codeid === 'coursecode') {
-			window.bool2 = true;
-		}
-
-	} else {
-
-		code.style.borderColor = "#E54";
-		x2.style.display = "block";
-		code.style.borderWidth = "2px";
-		if (codeid === 'ncoursecode') {
-			window.bool = false;
-		}
-		if (codeid === 'coursecode') {
-			window.bool2 = false;
-		}
-
-	}
-
-
-}
-/*Validates all forms*/ 
-function validateForm(formid) {
-
-  
-	if (formid === 'newCourse') {
-		var nName = document.getElementById("ncoursename").value;
-		var nCode = document.getElementById("ncoursecode").value;
-		//if inputs are empty
-		if (nName == null || nName == "", nCode == null || nCode == "") {
-			alert("Fill in all fields");
-
-		}
-    //if coursecode and coursenames are correct
-		if (window.bool3 === true && window.bool === true) {
-			alert('The course is now added');
-			createNewCourse();
-			$('#newCourse input').val("");
-
-
-		} else {
-			alert("You have entered incorrect information");
-		}
-	}
-	if (formid === 'editCourse') {
-		var Name = document.getElementById("coursename").value;
-		var Code = document.getElementById("coursecode").value;
-
-		//if inputs are empty
-		if (Name == null || Name == "", Code == null || Code == "") {
-			alert("Fill in all fields");
-
-		}
-    //if coursecode and coursenames are correct
-		if (window.bool4 === true && window.bool2 === true) {
-			alert('The course updated');
-			updateCourse();
-			
-
-
-		} else {
-			alert("You have entered incorrect information");
-		}
-	}
-}
```

Comment:
How are the deletion of these row a part of this issues solution?

Reply:
Hmm, I'm not sure why that commit shows under this issue. It seems to belong in https://github.com/HGustavs/LenaSYS/issues/7250